src/components/OpenSeadragonViewer.js

```diff
@@ -37,6 +38,15 @@ class OpenSeadragonViewer extends Component {
     }, 3000);
   };
 
+  canDownloadFullSize() {
+    const { rightsStatement } = this.props;
+
+    return (
+      rightsStatement.hasOwnProperty('label') &&
+      rightsStatement.label.indexOf('No Copyright') > -1
```

Comment:
I think this would allow access with more rights statements than the one Jen specified? (There are several that contain 'No Copyright')

https://github.com/nulib/donut/blob/master/config/authorities/rights_statements.yml

Reply:
@kdid Correct, this will let any "No Copyright..." value pass.  Guess I mis-read or mis-interpreted the comments, or assumed that any "No Copyright..." value would work.  I'll update and re-push.